### DISCUSSION TITLE
Sabotage State as entity

### DISCUSSION THREAD

**Discussion by williknow** [SukramJ/aiohomematic#1407] [Q&A] [answered]
Hello,
I have a Homematic HM-Sec-SC-2. This device has a sabotage contact which with I would like to trigger an automation in homeassistant. In the logbook I can see "error 7" if the contact is open and "errors solved" if the contact is closed. Can we make an entity of that? I have seen in the code of hahomatic the errors are hidden? Is there an option to unhide?

**Comment by SukramJ** [comment, marked as answer]
Interesting. You read the code, but not the [docs](https://github.com/danielperna84/custom_homematic/tree/devel#homematicip-local-documentation).

Option 1: Use the [Events](https://github.com/danielperna84/custom_homematic/tree/devel#homematicdevice_error). There is also a [blueprint](https://github.com/danielperna84/custom_homematic/blob/devel/blueprints/automation/homematicip_local_show_device_error.yaml).
Option 2: Use [unignore](https://github.com/danielperna84/custom_homematic/tree/devel#unignore-device-parameters) and add ERROR

**Reply by williknow** [reply]
Thank you for your answer.
Option one was to complicated for me.
I used unignore and it worked as desired.
